### ISSUE TITLE
Error using VertexTangentsHelper, doesn't occur in three/examples

### ISSUE THREAD

**Issue by mattrossman** [pmndrs/three-stdlib#62]
- `three` version: r129
- `three-stdlib` version: 2.0.5

### Problem description:

`VertexTangentsHelper` plugged into drei's `useHelper` causes an error, while the corresponding helpers in `three/examples` work as expected

### Relevant code:

https://codesandbox.io/s/oiix2

Commenting out the `three/examples` import and uncommenting the `three-stdlib` one produces the error:
```
ReferenceError
Must call super constructor in derived class before accessing 'this' or returning from derived constructor
```

### Suggested solution:

null

It's possible that this is actually an issue with `useHelper` but I expect that three-stdlib should work the same three/examples.

**Comment by joshuaellis** [comment]
Thanks for raising this, i can see from our file that we're not calling `super` a.s.a.p which is odd... 

https://github.com/pmndrs/three-stdlib/blob/4d561a22a30c96224eff149bf46d0c1c757b9455/src/helpers/VertexTangentsHelper.js#L33

Do you know when they call `super` in the three version?

**Comment by mattrossman** [comment]
Looks to be in a similar spot, they don't call it immediately either:
https://github.com/mrdoob/three.js/blob/dev/examples/jsm/helpers/VertexTangentsHelper.js#L35

**Comment by joshuaellis** [comment]
ah, except they're not accessing `this` before hand. This is a simple fix. give me 5.

**Comment by mattrossman** [comment]
Thanks for the update. Unfortunately, I'm still seeing the issue in version 2.0.6:

https://codesandbox.io/s/vertex-tangents-bug-forked-f8zi0?file=/src/App.js

**Comment by joshuaellis** [comment]
Oops thanks @mattrossman, my bad! I missed one of the calls to `this`, all fixed in 2.0.7 now though – https://codesandbox.io/s/vertex-tangents-bug-forked-ouwdb
